### DISCUSSION TITLE
No mobile phone, how to do 2FA?

### DISCUSSION THREAD

**Discussion by MeatAcorn** [community/community#133009] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The recent message about forcing 2FA mentions several things about mobile apps. I have a land line, no SMS or apps or anything like that. Need to disable 2FA or allow another way, Microsoft doesn't need my real phone number either.

**Comment by romhenri** [comment]
Have you found this?

https://docs.github.com/pt/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication

**Comment by MeatAcorn** [comment]
Yes, it wants me to buy a mobile phone or install something on my desktop, silly when I have the correct username and password. I do work, don't play on phones. Plus if it's a desktop app, it means I can only log in one place and I have Coder environments that don't have desktops, this isn't a good solution.

Thanks.

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by knarrff** [comment]
You can try this: https://www.cyberciti.biz/faq/use-oathtool-linux-command-line-for-2-step-verification-2fa/. It works on the command line and is essentially the same that would happen on a phone.

**Comment by aishamc** [comment]
If you use a landline and need to deal with two-factor authentication (2FA) but can't use SMS or an app, here’s what you can do:

Disable 2FA: Sign in to your Microsoft account and go to the security settings. Look for an option to turn off two-factor authentication and follow the steps to disable it.

Alternative Verification: Instead of using SMS or an app, you can set up email-based verification. Check your security settings to see if you can use your email for verification. If you have backup codes from when you set up 2FA, you can use those to sign in.

Update Contact Info: Make sure your account has a current email address. Some services might let you use a landline for verification, so check if that option is available.

If you have trouble or these steps don’t work, contact Microsoft support for help. They can guide you through resolving 2FA issues.
